Code/Framework/AzCore/AzCore/UnitTest/UnitTest.h

```diff
@@ -260,6 +317,28 @@ namespace UnitTest
         bool m_createdAllocator = false;
     };
 
+    inline TraceBusHookUniqueDisableToken::TraceBusHookUniqueDisableToken(TraceBusHook* traceBusHook)
```

Comment:
This looks like a scoped class, why not call it ScopeTraceBusDisable or something simple like that?